wrapper/src/main/java/software/amazon/jdbc/plugin/failover/FailoverConnectionPlugin.java

```diff
@@ -461,6 +419,7 @@ protected void updateTopology(final boolean forceUpdate) throws SQLException {
    * @return true if the given method is allowed on closed connections
    */
   private boolean allowedOnClosedConnection(final String methodName) {
+    // TODO: consider to use target driver dialect
```

Comment:
Is this `TODO` intended to be done by this PR or a future one?

Reply:
That's for the future PR. I don't want to mix scope of the current PR.